CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Changelog
+
+## 1.32.0
+* Feature - Add support for automatic spot instance draining [#2205](https://github.com/aws/amazon-ecs-agent/pull/2205)
+* Bug - Initialize ENI ack timer if needed upon restart.
```

Comment:
can you use this one - Fixed a bug where the agent might crash if it's restarted right after launching a task in awsvpc network mode #2219 (in the "Description for the changelog section" of the pr)

Reply:
done